lib/application.js

```diff
@@ -49,21 +49,28 @@ class Application extends events.EventEmitter {
 
   async createScript(fileName) {
     const code = await fsp.readFile(fileName, 'utf8');
-    const src = '\'use strict\';\n' + code;
+    const src = '\'use strict\';\ncontext => ' + code;
     const options = { filename: fileName, lineOffset: -1 };
     try {
-      return new vm.Script(src, options);
+      const script = new vm.Script(src, options);
+      return script.runInContext(this.sandbox, SCRIPT_OPTIONS);
     } catch (err) {
       this.logger.error(err.stack);
       return null;
     }
   }
 
-  runScript(methodName, session) {
-    const { sandbox } = session || this;
+  runScript(iname, ver, methodName, session) {
```

Comment:
Do not change this method signature because we have no multiple interfaces with multiple versions implemented in starter kit, we use just method names, we can't backport all this stuff without changes.